waku/v2/protocol/filter/waku_filter.go

```diff
@@ -193,9 +194,11 @@ func (wf *WakuFilter) FilterListener() {
 	handle := func(envelope *protocol.Envelope) error { // async
 		msg := envelope.Message()
 		topic := envelope.PubsubTopic()
+		g, _ := errgroup.WithContext(context.Background())
```

Comment:
If you don't want the context then you can create the group with just `new`, see https://pkg.go.dev/golang.org/x/sync/errgroup#example-Group-JustErrors

Reply:
Thanks. Don't know how I missed that.